src/main/java/commands/WorkoutCommand.java

```diff
@@ -119,7 +123,8 @@ public void execute() {
         } catch (NumberFormatException e) {
             System.out.println("Uh oh, a number was expected in your input, but a non-formattable\n"
                     + "number was received.");
-            System.out.println("Please try again.");
+            System.out.println("Please try again.Alternatively, type 'help' if you need\n"
```

Comment:
Loved that follow up action is added! Just a minor minor formatting issue - add a space in between "again." and "Alternatively"! 👍